tasks/build.js

```diff
@@ -0,0 +1,79 @@
+// @ts-check
+const bundleAPI = require('./bundle-api');
+const bundleCSS = require('./bundle-css');
+const bundleHTML = require('./bundle-html');
+const bundleJS = require('./bundle-js');
+const bundleLocales = require('./bundle-locales');
+const clean = require('./clean');
+const copy = require('./copy');
+const reload = require('./reload');
+const {runTasks} = require('./task');
+const {log} = require('./utils');
+
+async function release() {
+    log.ok('RELEASE');
+    try {
+        await runTasks(standardTask, {production: true});
```

Comment:
Some release tasks are missing, I suggest you to add them like `[...standardTasks, codeStyle, zip]`